pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -224,6 +224,76 @@ def _set_dimensional_parameters(self):
             "Initial concentration in electrolyte [mol.m-3]"
         )
 
+        # mechanical prapmeters
+        if self.options["particle cracking"] is not None:
+            self.flag_mechanics = pybamm.Parameter("Flag mechanical effects")
+            # flag: 1 - activate mechanical effects; 0 - disable
+            self.nu_p = pybamm.Parameter("Positive electrode Poisson's ratio")
+            self.E_p = pybamm.Parameter("Positive electrode Young's modulus [Pa]")
+            self.c_p_0 = pybamm.Parameter(
+                "Positive electrode reference concentration for free of deformation "
+                "[m3.mol-1]"
+            )
+            self.Omega_p = pybamm.Parameter(
+                "Positive electrode partial molar volume [m3.mol-1]"
+            )
+            self.nu_n = pybamm.Parameter("Negative electrode Poisson's ratio")
+            self.E_n = pybamm.Parameter("Negative electrode Young's modulus [Pa]")
+            self.c_n_0 = pybamm.Parameter(
```

Comment:
should units be [mol.m-3]? same for `c_p_0`

Reply:
corrected for both